policies/committer-policy.md

```diff
@@ -0,0 +1,102 @@
+
+# Policy for OpenSSL Committers
+
+## Who is a committer?
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.  Committers review and commit their own patches
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by the [OMC] typically on the recommendation
+of the [OTC] (see the [OpenSSL Bylaws]).
+
+We welcome contributors who become domain experts in some part of
+the library (for example, low-level crypto) as well as generalists
+who contribute to all areas of the codebase.  All committers share
+the responsibility for the overall health of the project: aside from
+contributing quality features, committers are team players who fix bugs,
+address open issues, review community contributions, and improve tests
+and documentation.  Committers are also shepherds of the OpenSSL community
+and its code of conduct.
+
+To become a committer, start by contributing code.  Read our coding style,
+and get to know our build and test system.  Then, use the Github issue
+tracker, and our mailing lists find impactful ideas to work on.
+
+## How to maintain committer status?
+
+To maintain committer status, you must stay active in the project.  Refer
+to the OpenSSL Bylaws for details.
+
+In the unlikely and unfortunate event that your actions conflict with
+the project objectives or are otherwise disruptive, committer status
+may also be revoked by the OMC.
+
+## Approvals and code reviews
+
+All submissions must be reviewed and approved by at least two committers,
+one of whom must also be an OTC member.  Neither of the reviewers can
+be the author of the submission.
+
+The sole exception to this is for the release procedure commits where
+the author's review does count towards the two needed.
+
+In the case where two committers make a joint submission, they can review
+each other's code but not their own.  A third reviewer will be required.
```

Comment:
So, to be clear, does this mean that in the case where a single PR contains commits from 2 different committers, we only need one additional reviewer? The 2 authors each review the commits authored by the other person, and a 3rd committer reviews all of them?

Reply:
Yes.  Or a 4th also reviews everything (in which case there is no issue at all).

I had trouble capturing this intent without creating a word jam of undue complexity.

Reply:
I'm not sure that this exception actually makes sense in practical terms. It would require a clearer view of who does what and a separate set of reviews for each commit and in practical terms that doesn't really seem to happen in PRs in general. I would leave this whole concept out and always have two reviewers who are not the author(s) and see how that works out in practical terms. i.e. this is a premature optimisation for a problem we haven't yet experienced with the changed review rule. We should IMHO put in the simple rule and adjust based on experience with the new rule over time.

Reply:
I'm in total agreement that wording isn't ideal as it currently stands.

I also believe that two completely independent reviewers will be problematic in pretty much any instance where this clause applies.

Reply:
I don't actually see this changing anything, 3 people are needed for the normal case and this case.

Reply:
I'm with @paulidale that to require 2 additional people to review something done by joint work of 2 committers would be seriously problematic. And I do not think the current wording is unclear or anything.